packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -0,0 +1,239 @@
+import '../ui/plotly.dart';
+
+class FramesBarPlotly {
+  static const int CPU_GOOD_TRACE = 0;
+  static const int  CPU_JANK_TRACE = 1;
```

Comment:
run dartfmt on the code base to fix formatting nits.

Reply:
Done.